HoloRepositoryUI/client/src/services/holoRepositoryServerService.ts

```diff
@@ -0,0 +1,121 @@
+import serverAxios, { routes } from "./holoRepositoryServerAxios";
+import {
+  IPatient,
+  IPractitioner,
+  IHologram,
+  IImagingStudy,
+  IPipeline
```

Comment:
It's fine, but just curious why there is an "I" in front of the class name? any special purpose?

Reply:
It's just a naming convention for TypeScript interfaces (so the I is for interface). There are different opinions on whether or not to do this, but I think as long as we do it consistently it is fine.

Reply:
OK, it's fine, I have done the review for the code, will merge it, nice work!